apps/web/layouts/PageLayout/AuthLayout/NavLink.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from 'clsx'
+import Link from 'next/link'
+import {useRouter} from 'next/router'
+import type {ComponentType} from 'react'
+
+import type {SvgIconProps} from '../../../components/icons'
+
+export interface NavLinkProps {
+  name: string
+  href: string
+  icon: ComponentType<SvgIconProps>
+}
+
+export function NavLink({name, href, icon: Icon}: NavLinkProps) {
```

Comment:
@iwasrobbed active link looks like this, wdyt?
<img width="427" alt="image" src="https://user-images.githubusercontent.com/3942264/215315432-bf21258e-c466-4b14-bee7-bf9ccd005da7.png">

Reply:
Looks great!  I'll have a closer look once the Vercel deploy is fixed and I can see it in use

The only 🤔 is the left nav looks wider than in the designs so we may want to tweak that to be more narrow (again something we can look at once deployed)

Reply:
Good eye! Sorry I forgot to update that one. Corrected 538331e